cfn-resources/cloud-backup-schedule/cmd/resource/model.go

```diff
@@ -74,15 +60,15 @@ type ApiPolicyItemView struct {
 	RetentionUnit     *string `json:",omitempty"`
 }
 
-// ApiKeyDefinition is autogenerated from the json schema
-type ApiKeyDefinition struct {
-	PublicKey  *string `json:",omitempty"`
-	PrivateKey *string `json:",omitempty"`
-}
-
 // Link is autogenerated from the json schema
 type Link struct {
 	ApiKeys *ApiKeyDefinition `json:",omitempty"`
 	Href    *string           `json:",omitempty"`
 	Rel     *string           `json:",omitempty"`
 }
+
+// ApiKeyDefinition is autogenerated from the json schema
+type ApiKeyDefinition struct {
```

Comment:
Is this still needed?

Reply:
This APIKeys is different, this is from Link Property